openshift/templates/agent/agent-deploy.param

```diff
@@ -27,7 +28,7 @@ LOG_LEVEL=WARNING
 TAILS_FILES_DIR=/home/indy/.indy_client/tails
 TAILS_FILES_VOLUME_CLASS=netapp-file-standard
 TAILS_FILES_VOLUME_SIZE=1Gi
-TAILS_SERVER_BASE_URL=https://tails-server-dev.pathfinder.gov.bc.ca
+TAILS_SERVER_BASE_URL=https://tails-server-dev.apps.silver.devops.gov.bc.ca
```

Comment:
As we go through the deployments, it might be a good time to update the tails server urls to the `*.vonx.io` ones.

Reply:
change completed